coral-incremental/src/main/java/com/linkedin/coral/incremental/IncrementalTransformerResults.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.incremental;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.calcite.rel.RelNode;
+
+
+public class IncrementalTransformerResults {
```

Comment:
I think the code will be much simpler if we get rid of this class, and move the members to the visitor itself.